### PR TITLE
add support for macOS

### PR DESCRIPTION
**Pull request by salamaashoush**

### CODE REVIEW

File: src/threepp/renderers/gl/GLCapabilities.hpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ namespace threepp::gl {
 
     private:
         GLCapabilities()
-            : maxAnisotropy(glGetParameter(GL_MAX_TEXTURE_MAX_ANISOTROPY)),
+            : maxAnisotropy(0),
```
Comment (salamaashoush) [inline src/threepp/renderers/gl/GLCapabilities.hpp:62]: this one is not supported for gl 4.1

**Comment by markaren** [comment]
Mhh, GitHub Actions should have been triggered by this PR.

The changes looks ok. Will have a closer look later today.

File: src/threepp/renderers/gl/GLCapabilities.hpp (hunk 1 above)
Comment (markaren) [reply src/threepp/renderers/gl/GLCapabilities.hpp:62]: This flag has very limited usage in the [three.js codebase](https://github.com/mrdoob/three.js/search?q=maxAnisotropy), so we can probably remove it altogether

**Comment by markaren** [comment]
Aight, it now compiles on OSX 10.5 using the GitHub CI. Merging this.

**Pull request merged**